dhcpv4/dhcpv4.go

```diff
@@ -248,6 +248,23 @@ func NewRequestFromOffer(offer *DHCPv4, modifiers ...Modifier) (*DHCPv4, error)
 	)...)
 }
 
+// NewInformFromOffer builds a new DHCPv4 Informational message from the given
+// offer with the specified hardware address.
+func NewInformFromOffer(offer *DHCPv4, modifiers ...Modifier) (*DHCPv4, error) {
+	xid, err := GenerateTransactionID()
```

Comment:
I think this can be removed. It is called in the `New` below.

Reply:
Good catch, fixed.